descriptors/ModuleDescriptor-template.json

```diff
@@ -35,9 +35,9 @@
             "mod-settings.entries.item.post"
           ],
           "permissionsDesired": [
-            "mod-settings.global.write.*",
-            "mod-settings.users.write.*",
-            "mod-settings.owner.write.*"
```

Comment:
What is the motivation for this and similar changes?

Reply:
In order to permit permission schemes that work with Eureka.

Reply:
Ah yes, of course.